### PR TITLE
User can view a list of lodges. #1

### PR DESCRIPTION
**Pull request by kevinhyder**
screen shot 2017-06-15 at 2 47 36 pm

### CODE REVIEW

**Comment by kevinhyder** [comment]
screen shot 2017-06-15 at 10 22 04 am

File: main.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    city: 'Anaheim ',
+    state: 'CA, ',
+    zip: '92805',
+    phone: '(714) 543-7277 ',
+    website: 'www.ocscottishrite.org',
+    body: 'Scottish Rite',
+    day: 'Monday'
+  }
+]
+
```
Comment (thebearingedge) [inline main.js:63]: Good work, let's talk about your rendering strategy though. There's a lot of repetition here. We can talk about leveraging a loop to do this work.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (thebearingedge) [reply main.js:63]: - [x] Remove all HTML for individual lodges (you'll create it from JavaScript).
- [x] Define a function that takes a lodge and renders a DOM element of the lodge.
- [x] Loop through lodges and for each one, render it, and append it to the desired location on the page.

**Review by thebearingedge — APPROVED** [review]
Good job, feel free to merge this.

**Pull request merged**